app/components/badge-maintenance.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+    tagName: 'span',
+    classNames: ['badge'],
+    escapedStatus: computed('badge', function() {
+        return this.get('badge.attributes.status').replace('-', '--');
+    }),
+    none: computed('badge', function() {
+        if (this.get('badge.attributes.status') === 'none'
+            || this.get('badge.attributes.status') == undefined) {
+            return true;
+        }
+        return false;
```

Comment:
I think this function could be simplified by returning the value from the comparisons directly, wdyt? 

```
return this.get('badge.attributes.status') === 'none' || !this.get('badge.attributes.status');
```

Reply:
Hmm I'm not quite sure what `!this.get('badge.attributes.status')` does, I mean if it gets the status attribute then it is a not against it? Does the statement return true if the value is `undefined`, and false if there is a string returned?

Reply:
I decided to change it since it works the same anyway :). Should I add a comment explaining this intruiging JavaScript behaviour?

Reply:
Most people experienced with javascript have had experience with how it treats true/false/undefined behaviors, so I don't think this deserves a comment, no.